classes/class-wc-emspay-helper.php

```diff
@@ -44,6 +44,30 @@ class WC_Emspay_Helper
      */
     public static $afterPayCountries = ['NL', 'BE'];
 
+    /**
+     * @var array
+     */
+    public static $supportedCurrencies = ['EUR'];
+
+    /**
+     * @var array
+     */
+    public static $supportedGatewayCurrencies = [
+        'emspay_ideal' => ['EUR'],
+        'emspay_bank-transfer' => ['EUR'],
+        'emspay_credit-card' => ['EUR', 'GBP', 'USD', 'CAD', 'CHF', 'CZK', 'DKK', 'NOK', 'PLN', 'SEK', 'AUD', 'HKD', 'JPY', 'NZD', 'SGD', 'ZAR', 'BRL', 'TRY'],
+        'emspay_bancontact' => ['EUR'],
+        'emspay_klarna-pay-now' => ['EUR', 'SEK', 'CHF'],
+        'emspay_paypal' => ['AUD', 'CAD', 'CHF', 'CZK', 'DKK', 'EUR', 'GBP', 'HKD', 'NOK', 'PLN', 'SEK', 'SGD', 'USD'],
+        'emspay_klarna-pay-later' => ['EUR', 'DKK', 'NOK', 'SEK', 'CHF', 'GBP'],
```

Comment:
The currency depends on the country for Klarna
	https://developers.klarna.com/documentation/klarna-payments/in-depth-knowledge/puchase-countries-currencies-locales/